### PR TITLE
Create DEVELOPERS.md

### PR DESCRIPTION
**Pull request by codeShaurya**
# Issue #57 
Created a new file named DEVELOPERS.md 

### Changes
- I created a new file in the root folder and fixes the 404 bug

- Created DEVELOPERS.md file

### Related Issues
- Issue #57

### CODE REVIEW

**Comment by irmerk** [comment]
I believe this is already currently being worked on in PR #58. If @vaibhav111tandon closes that PR or doesn't respond in a few days, we can review this PR.

**Comment by codeShaurya** [comment]
Sure @irmerk

File: DEVELOPERS.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+cd <repository>
+
+# Add the main repository as an upstream remote to your repository:
+git remote add upstream "https://github.com/acccordproject/<repository>.git"
+
+# Install node.js dependencies:
+npm install
+```
+
+### Running the Project Locally
```
Comment (jeromesimeon) [inline DEVELOPERS.md]: This does not seem to apply to this project. Please review and update the `DEVELOPERS.md` file so it reflects this project.

**Comment by vaibhav111tandon** [comment]
Already being updated in PR #58

**Comment by jeromesimeon** [comment]
Already fixed. I'm closing this PR.

**Pull request closed without merging**